projector-agent-common/src/test/kotlin/org/jetbrains/projector/agent/common/DummyTest.kt

```diff
@@ -23,12 +23,13 @@
  */
 package org.jetbrains.projector.agent.common
 
-import kotlin.test.Test
-import kotlin.test.assertTrue
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.booleans.shouldBeTrue
 
-class MiscTest {
-  @Test
-  fun `empty test for coverage building`() {
-    assertTrue(true)
+class DummyTest : StringSpec() {
+  init {
+    "empty test for coverage building" {
+      true.shouldBeTrue()
+    }
```

Comment:
```suggestion
    "empty test for coverage building" {}
```

As discussed, can we just remove the assertion (here and in other places)?